CWE/localisation/text.csv

```diff
@@ -13838,16 +13838,10 @@ TAJ_populist;Patriot Party;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
 TAJ_progressive;Libertarian Party;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
 TAJ_socialist;Labour Party;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
 TAKE_SIDE_PENALTY;Prestige penalty for not taking a side: �R$VAL$�!;;Malus auf Prestige, da keine Seite gew�hlt wurde: �R$VAL$�!;;;;;;;;;;;x;;;;;;;;;;;;;;;;
-TAN;Tanzania;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
-TAN_colonial_government;Tanganyika;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
-TAN_mandate_government;Tanganyika;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
-TAN_hms_government;Tanganyika;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
-TAN_hms_government1;Tanganyika;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
-TAN_colonial_government_ADJ;Tanganyikan;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
-TAN_mandate_government_ADJ;Tanganyikan;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
-TAN_hms_government_ADJ;Tanganyikan;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
-TAN_hms_government1_ADJ;Tanganyikan;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
-TAN_ADJ;Tanzanian;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
+TAN;Tanganyika;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
+TAN_democracy1;Tanzania;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
+TAN_democracy1_ADJ;Tanzanian;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
+TAN_ADJ;Tanganyikan;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
```

Comment:
Okay i think the better way is just to create a separate tag for Tanzania

Reply:
Oh I thought it would have been an issue to create new tags

Reply:
In this situation, the best approach is just to make new tags as otherwise the name will always have a chance of not matching the in game situation.